utils/unique_email_enforcement.ts

```diff
@@ -10,11 +10,3 @@ export const profileEmailTableClient = TableClient.fromConnectionString(
   config.PROFILE_EMAIL_STORAGE_CONNECTION_STRING,
   config.PROFILE_EMAIL_STORAGE_TABLE_NAME
 );
-
-export const FF_UNIQUE_EMAIL_ENFORCEMENT_ENABLED = getIsUserEligibleForNewFeature<
-  FiscalCode
->(
-  fiscalCode => config.UNIQUE_EMAIL_ENFORCEMENT_USERS.includes(fiscalCode),
-  () => false,
-  config.FF_UNIQUE_EMAIL_ENFORCEMENT
-);
```

Comment:
The `config.UNIQUE_EMAIL_ENFORCEMENT_USERS` variable was previously used to define beta testers for the "Unique Email Enforcement" feature. 
Since the feature flag has been removed, this variable is no longer necessary and can be safely deleted.

Reply:
Solved in 0ab49e33b68c0796f550370db4fe37c971c099e6